### PR TITLE
Sync cluster app

### PR DESCRIPTION
**Pull request by natifridman**
Add a task to refresh the gitops-controller app and wait for the new cluster app to sync.

Depends on https://github.com/redhat-partner-solutions/vse-carslab-hub/pull/61

### CODE REVIEW

File: components/tekton/pipelineruns/create-cluster/create-cluster-pipelinerun.yaml [hunk 1]
```diff
@@ -26,7 +26,10 @@ spec:
     - name: hub_git_revision
       value: main
     - name: hub_git_url
-      value: 'git@github.com:redhat-partner-solutions/vse-carslab-hub.git'
+      # value: 'git@github.com:redhat-partner-solutions/vse-carslab-hub.git'
+      value: 'git@github.com:natifridman/vse-carslab-hub.git'
```
Comment (novacain1) [inline components/tekton/pipelineruns/create-cluster/create-cluster-pipelinerun.yaml]: Should this be your fork or the redhat-partner-solutions repo?

**Review by novacain1 — COMMENTED** [review]
Looks good, other than one comment.

File: components/tekton/pipelineruns/create-cluster/create-cluster-pipelinerun.yaml (hunk 1 above)
Comment (natifridman) [reply components/tekton/pipelineruns/create-cluster/create-cluster-pipelinerun.yaml]: Oops, this is leftover from my tests. Fixed

**Review by hhamid93 — COMMENTED** [review]
Looks good to me.

**Review by hhamid93 — APPROVED** [review]

**Pull request merged**